packages/neuron-ui/src/widgets/Table/index.tsx

```diff
@@ -57,6 +59,37 @@ const Table = <T extends Record<string, any>>(props: TableProps<T>) => {
 
   const columnList = useMemo(() => columns.filter(item => !item.hidden), [columns])
 
+  const [sortIndex, setSortIndex] = useState(-1)
+  const [sortType, setSortType] = useState('')
+
+  const currentDataSource = useMemo(() => {
+    if (sortIndex !== -1 && sortType) {
+      const { sorter } = columnList[sortIndex]
+      if (sorter) {
+        const list = [...dataSource].sort(sorter)
+        return sortType === 'increase' ? list : list.reverse()
```

Comment:
It's better to define `increase` and `decrease` into enum or const variable.

Reply:
And it's better to sort in `sort`, but not use reverse. Because there may exist some strange order.
For example, some table rows should stay first always. Then the result order is controlled by users but not in the table component.